cvat/apps/engine/views.py

```diff
@@ -785,6 +785,12 @@ class TaskViewSet(viewsets.GenericViewSet, mixins.ListModelMixin,
         'subset', 'mode', 'dimension', 'tracker_link'
     )
     filter_fields = list(search_fields) + ['id', 'project_id', 'updated_date']
+    filter_description = dedent("""
+
+        Examples:
+        Get all tasks from 1,2,3 projects: {"and":[{"in":[{"var":"project_id"}, [1, 2, 3]]}]}
+        Get all tasks created by me: {"and":[{"==":[{"var":"owner"},"<user>"]}]}
+    """)
```

Comment:
Probably, it would be better to add a default description in the filter class, so all related viewsets automatically get a description for the filter param. Not just only the tasks viewset.

We should probably add this explanation into the docs as well and answer several related issues by doing this.

Reply:
![image](https://github.com/opencv/cvat/assets/49038720/87990db7-6cdd-409d-8222-b7dcae63e1bf)